influxdb_connector/influxdb_connector.py

```diff
@@ -89,37 +137,38 @@ def custom_bucket_formatting(csv_builder, bucket):
         A dictionary containing time and field columns from influxdb query
     """
     # custom formatting for anomaly detection and time series bucket
```

Comment:
Could you clarify why formatting depends on bucket name? 

Wonder if possible to decouple these ?

Reply:
I've tried multiple setups to decouple this but the only other way I was able to find was to require users to list all the column name as input. I'll leave as it is for now and update if that's the direction we would like to go.